crates/syntax/src/ast/node_ext.rs

```diff
@@ -725,6 +725,23 @@ impl ast::TokenTree {
     pub fn parent_meta(&self) -> Option<ast::Meta> {
         self.syntax().parent().and_then(ast::Meta::cast)
     }
+
+    pub fn as_comma_sep_paths(self) -> Option<Vec<ast::Path>> {
+        let r_paren = self.r_paren_token()?;
+        let tokens = self
+            .syntax()
+            .children_with_tokens()
+            .skip(1)
+            .take_while(|it| it.as_token() != Some(&r_paren));
+        let input_expressions = tokens.into_iter().group_by(|tok| tok.kind() == T![,]);
+        Some(
+            input_expressions
+                .into_iter()
+                .filter_map(|(is_sep, group)| (!is_sep).then(|| group))
+                .filter_map(|mut tokens| ast::Path::parse(&tokens.join("")).ok())
+                .collect::<Vec<ast::Path>>(),
+        )
+    }
```

Comment:
You already correctly pointed out, this should go into `ide_db` somewhere instead. We try to not do any transformations in `node_ext`.